.github/workflows/snyk-pr.yml

```diff
@@ -6,6 +6,7 @@ on:
 jobs:
   whitesource:
     runs-on: ubuntu-latest
+    permissions: write-all
```

Comment:
what's the issue here? normally prod-sec take care of snyk.

Reply:
When troubleshooting .NET Snyk PR [#690](https://github.com/snowflakedb/snowflake-connector-net/pull/690) we have found that Snyk did not have the permissions to comment/modify the title of the PR thus the `snyk-pr / whitesource` job constantly failed.

Adding this permissions resolved the issue.

Reply:
do we need proc-sec approval on this PR?

Reply:
This change has been copied from their change in `snowflake-connector-net` driver.

Reply:
@sfc-gh-jfan please check.

Reply:
it looks good to me, it should be able to fix the issue here